server/api/api.go

```diff
@@ -29,6 +29,9 @@ type Subscriptions interface {
 
 type Calendar interface {
```

Comment:
1. As a general comment - more naming consistency and intention-revealing, typographically pleasing names would be nice.
2. Support for specific calendars is currently out of scope for 1.0. 1/5 we should accept the functionality in this PR for future use and/or debugging; think about restricting the commands to "debug-only" at some point.